lib/reporters/index.js

```diff
@@ -21,6 +21,10 @@ internals.protos = {
 
 internals.requireReporter = function (reporter) {
 
+    if (Array.isArray(reporter)) {
+        return require('./multiple');
```

Comment:
Add a comment to the top of the file that indicates that multiple is required further down.  Just easier to look at the module section to find dependencies

```js
// ('./multiple') loaded below
```